VisualRust.Test/Build/RustcMessageJsonParserTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using VisualRust.Build.Message;
+using VisualRust.Build.Message.Json;
+
+namespace VisualRust.Test.Build
+{
+    class RustcMessageJsonParserTest
+    {
+        [Test]
+        public void WithoutError()
+        {
+            const string output = "";
+            var messages = RustcMessageJsonParser.Parse(output).ToList();
+
+            Assert.AreEqual(0, messages.Count);
+        }
+
+        [Test]
+        public void FileNotFound()
+        {
+            const string output = "{\"message\":\"couldn't read \\\"main2.rs\\\": Не удается" +
```

Comment:
Does this mean that the test will fail when the system language is not Russian? That would be bad. Can you try to make this locale independent?